src/webapp/CompositionRoot.ts

```diff
@@ -116,6 +125,9 @@ export async function getCompositionRoot(instance: Instance) {
         analytics: getExecute({
             sendPageView: new SendPageViewUseCase(analyticsRepository, configRepository),
         }),
+        matomo: getExecute({
```

Comment:
In my opinion, I would add `trackView` under analytics as I see both GA and Matomo as analytics.